docs-src/concepts/what-is-a-codec-server.md

```diff
@@ -24,6 +24,7 @@ Most SDKs provide example Codec Server implementation samples, listed here:
 
 - [Go](https://github.com/temporalio/samples-go/tree/main/codec-server)
 - [Java](https://github.com/temporalio/sdk-java/tree/master/temporal-remote-data-encoder)
+- [.Net](https://github.com/temporalio/samples-dotnet/tree/main/src/Encryption)
```

Comment:
I think this is supposed to be ".NET" generally but definitely not worth blocking a small addition until we have an SDK

Reply:
Thanks, fixed!